simplivity/resources/external_stores.py

```diff
@@ -106,7 +106,8 @@ def register_external_store(self, management_ip, name, cluster, username, passwo
             cluster = clusters_obj.get_by_name(cluster)
 
         data['omnistack_cluster_id'] = cluster.data['id']
-        self._client.do_post(URL, data, timeout)
+        custom_headers = {'Content-type': 'application/vnd.simplivity.v1.14+json'}
```

Comment:
Is this the minimum supported content type?

Reply:
Yes this the minimum supported content-type for API

Reply:
Minimum version for the API is the ''application/vnd.simplivity.v1.11+json'', but in swagger the default version select is ''application/vnd.simplivity.v1.14+json'' therefore we used this version.

Reply:
Verified and tested with the minimum version ''application/vnd.simplivity.v1.11+json'' API works fine. Therefore updated the content-type to minimum version
Thanks @svt-kmcdaniel